basics/indexing/json-index.md

```diff
@@ -404,11 +404,40 @@ WHERE JSON_MATCH(person, '"$.name"=''adam''')
 Find all persons who have an address (one of the addresses) with number 112:
 
 ```sql
-SELECT ... 
-FROM mytable 
+SELECT ...
+FROM mytable
 WHERE JSON_MATCH(person, '"$.addresses[*].number"=112')
 ```
 
+Find all persons who have at least one address that is not in the US:
+
+```sql
+SELECT ...
+FROM mytable
+WHERE JSON_MATCH(person, '"$.addresses[*].country" != ''us''')
+```
```

Comment:
@Jackie-Jiang should we explicitly call out that this semantic is different in previous versions?

Reply:
We can add a note stating exclusive predicate doesn't work properly on array, and cannot be nested in the previous version

Reply:
It's not just the array based lookups, there's also the non-existent key case (5th point in this PR description - https://github.com/apache/pinot/pull/13139). I'm wondering how we can best phrase these differences in semantics? If we call out all the details with examples, it might end up being unnecessarily confusing to new users.